pkg/argocd/update_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/argoproj-labs/argocd-image-updater/pkg/tag"
 	"github.com/argoproj-labs/argocd-image-updater/test/fake"
 	"github.com/argoproj-labs/argocd-image-updater/test/fixture"
+	"gopkg.in/yaml.v2"
```

Comment:
To fix the lint error, looks like the above line `gopkg.in/yaml.v2` should be in its own group around line 11.

Reply:
@chengfang , thank you for your help. I already fixed it and pushed the change.

I'll mark the conversation as "resolved". Please, let me know if you find any further issue.